packages/components/src/SquiggleChart.tsx

```diff
@@ -123,10 +131,10 @@ export const SquiggleChart: React.FC<SquiggleChartProps> = (props) => {
 
           let total = 0;
           let cdf = sortedPoints.map((point: labeledPoint) => {
-            if (point.type == "discrete") {
+            if (point.type === "discrete") {
```

Comment:
I'm surprised that "==" made it into our codebase earlier. I guess we didn't have an auto-linter or something for TS.